main.go

```diff
@@ -5,60 +5,64 @@ import (
 	"errors"
 	"text/template"
 
+        "github.com/sensu-community/sensu-plugin-sdk/sensu"
+        "github.com/sensu/sensu-go/types"
 	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
-	corev2 "github.com/sensu/sensu-go/api/core/v2"
-	"github.com/sensu/sensu-plugins-go-library/sensu"
+
 )
 
 // Config holds configuration data for the handler
 type Config struct {
 	sensu.PluginConfig
 	APIToken        string
-	ChatID          uint64
+	ChatID          int64
```

Comment:
Is there a reason negative chat id's should be supported? I don't think they're supported by Telegram itself.

Reply:
gofmt ran, sorry about that.

As for negative chat Id's, they are for chat groups.

Reply:
Example using a negative ID

<img width="743" alt="Screen Shot 2020-06-02 at 2 13 24 PM" src="https://user-images.githubusercontent.com/5530473/83570860-a0282500-a4db-11ea-8896-efd24d43eb67.png">

Reply:
@Thor77 Just to echo, I need negative chat ids to be able to alert to a group chat - and there are a few others in issues #1 and #2 asking the same. Thanks to @nixwiz for working on this. Hoping you can get it merged swiftly, @Thor77 :-)

Reply:
Thanks for the explanation and sorry for ignoring those issues, I didn't get notifications for those because I wasn't "watching" the repo for some reason